ts/components/InstabugButtons.tsx

```diff
@@ -19,25 +19,53 @@ type Props = ReturnType<typeof mapStateToProps> &
   OwnProps &
   ReturnType<typeof mapDispatchToProps>;
 
-class InstabugButtonsComponent extends React.PureComponent<Props, {}> {
+type State = {
+  instabugReportType: "bug" | "chat" | "undefined";
```

Comment:
```suggestion
  instabugReportType: "bug" | "chat" | undefined;
```
or you can use option (fp-ts)

Reply:
I used Option, thanks !